compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1658,7 +1675,7 @@ rustc_queries! {
         desc { "optimization level used by backend" }
     }
 
-    query output_filenames(_: ()) -> Arc<OutputFilenames> {
+    query output_filenames(_: ()) -> &'tcx Arc<OutputFilenames> {
```

Comment:
🤣 amusing type, but I get why, we don't want to give `OutgoingCodegen` a `'tcx` lifetime